resources/meilisearch-release.md

```diff
@@ -113,9 +113,8 @@ The steps are the same as for `How to create the first pre-release (or RC i.e. R
 
 ⚙️ The CIs will be triggered to:
 - [Upload binaries](https://github.com/meilisearch/meilisearch/actions/workflows/publish-binaries.yml) to the associated GitHub release.
-- [Publish the Docker images](https://github.com/meilisearch/meilisearch/actions/workflows/publish-docker-images.yml) (`latest`, `vX.Y` and `vX.Y.Z`) to DockerHub -> check the "Docker meta" steps in the CI to check the right tags are created
+- [Publish the Docker images](https://github.com/meilisearch/meilisearch/actions/workflows/publish-docker-images.yml) (`vX.Y` and `vX.Y.Z`) to DockerHub -> check the "Docker meta" steps in the CI to check the right tags are created
```

Comment:
Why do you remove the `latest` docker tag? Our CI pushes it

Reply:
I put it back too.